ext/civicrm_admin_ui/managed/SavedSearch_Administer_Location_Types.mgd.php

```diff
@@ -0,0 +1,170 @@
+    'entity' => 'SavedSearch',
+    'cleanup' => 'always',
+    'update' => 'unmodified',
+    'params' => [
+      'version' => 4,
+      'values' => [
+        'name' => 'Administer_Location_Types',
+        'label' => E::ts('Administer Location Types'),
+        'form_values' => NULL,
+        'mapping_id' => NULL,
+        'search_custom_id' => NULL,
+        'api_entity' => 'LocationType',
+        'api_params' => [
+          'version' => 4,
+          'select' => [
+            'name',
+            'display_name',
+            'vcard_name',
+            'description',
+            'is_active',
+            'is_default',
+            'is_reserved',
+          ],
+          'orderBy' => [],
+          'where' => [],
+          'groupBy' => [],
+          'join' => [],
+          'having' => [],
+        ],
+        'expires_date' => NULL,
+        'description' => NULL,
+      ],
+    ],
+  ],
+  [
+    'name' => 'SavedSearch_Administer_Location_Types_SearchDisplay_Location_Types_Table',
+    'entity' => 'SearchDisplay',
+    'cleanup' => 'always',
+    'update' => 'unmodified',
+    'params' => [
+      'version' => 4,
+      'values' => [
+        'name' => 'Location_Types_Table',
+        'label' => E::ts('Location Types Table'),
+        'saved_search_id.name' => 'Administer_Location_Types',
+        'type' => 'table',
+        'settings' => [
+          'actions' => FALSE,
+          'limit' => 50,
+          'classes' => [
+            'table',
+            'table-striped',
+          ],
+          'pager' => [
+            'show_count' => TRUE,
+          ],
+          'placeholder' => 5,
+          'sort' => [
+            [
+              'name',
+              'ASC',
+            ],
+          ],
+          'columns' => [
+            [
+              'type' => 'field',
+              'key' => 'name',
+              'dataType' => 'String',
+              'label' => E::ts('Name'),
+              'sortable' => TRUE,
+              'title' => 'Location Type ID: [id]',
+            ],
+            [
+              'type' => 'field',
+              'key' => 'display_name',
+              'dataType' => 'String',
+              'label' => E::ts('Display Name'),
+              'sortable' => TRUE,
+              'editable' => TRUE,
+            ],
+            [
+              'type' => 'field',
+              'key' => 'vcard_name',
+              'dataType' => 'String',
+              'label' => E::ts('vCard'),
+              'sortable' => TRUE,
+              'editable' => TRUE,
+            ],
+            [
+              'type' => 'field',
+              'key' => 'description',
+              'dataType' => 'String',
+              'label' => E::ts('Description'),
+              'sortable' => TRUE,
+              'editable' => TRUE,
+            ],
+            [
+              'type' => 'field',
+              'key' => 'is_active',
+              'dataType' => 'Boolean',
+              'label' => E::ts('Enabled?'),
```

Comment:
In the other PRs we've been removing these rather silly question marks, so for consistency let's do it here too.
```suggestion
              'label' => E::ts('Enabled'),
```